app/main.py

```diff
@@ -1 +1,37 @@
-#  write your code here
+class Animal:
+    def __init__(self, name, appetite: int, is_hungry=True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
```

Comment:
remove unnecessary else

Reply:
how can I return 0 if removed that?
![Screenshot_133](https://user-images.githubusercontent.com/110389820/194341878-d84299c6-af4e-4aea-a009-9bb37553acfe.png)